website/docs/docs/deploy/ci-jobs.md

```diff
@@ -65,6 +65,10 @@ To make CI job creation easier, many options on the **CI job** page are set to d
 
    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/create-ci-job.png" width="90%" title="Example of CI Job page in the dbt Cloud UI"/>
 
+import GitProvidersCI from '/snippets/_git-providers-supporting-ci.md';
```

Comment:
i suggest placing the "Availability of features by Git provider" section right after "Prerequisites" but before "Set up CI jobs." that way, it balances visibility and relevance while maintaining a logical progression.

currently, i think it's placed in a weird place and has examples nested under it too

<img width="256" alt="Screenshot 2025-01-10 at 11 24 26" src="https://github.com/user-attachments/assets/fde5a0c4-6404-42ba-8b90-1144815d5fb6" />